app/models/manageiq/providers/openshift/container_manager_mixin.rb

```diff
@@ -32,10 +32,10 @@ def raw_connect(hostname, port, options)
     end
```

Comment:
I meant the latter, and regarding sub classing of our managers in manageiq I wrote I agree.